drivers/hwinfo/hwinfo_esp32.c

```diff
@@ -9,18 +10,29 @@
 #include <drivers/hwinfo.h>
 #include <string.h>
 
-ssize_t _impl_hwinfo_get_device_id(u8_t *buffer, size_t length)
+ssize_t z_impl_hwinfo_get_device_id(u8_t *buffer, size_t length)
 {
-	uint32_t fuse_rdata[] = {
-		sys_read32(EFUSE_BLK0_RDATA1_REG),
-		sys_read32(EFUSE_BLK0_RDATA2_REG),
-	};
+	u32_t rdata1 = sys_read32(EFUSE_BLK0_RDATA1_REG);
+	u32_t rdata2 = sys_read32(EFUSE_BLK0_RDATA2_REG);
+	u8_t id[6];
 
-	if (length > sizeof(fuse_rdata)) {
-		length = sizeof(fuse_rdata);
-	}
+	/* The first word provides the lower 32 bits of the MAC
+	 * address; the low 16 bits of the second word provide the
+	 * upper 16 bytes of the MAC address.  The upper 16 bits are
```

Comment:
Can you remove the double space after the dot (also in the next line)?

Reply:
I could, but this is a long-standing practice when using fixed-space fonts, and there are over 2100 examples of this in Zephyr.   If you want to block for this I'll change it, but I think this falls under personal preference.

There's interesting relevant discussion [here](https://en.wikipedia.org/wiki/Wikipedia_talk:Manual_of_Style/Archive_(spaces_after_a_full_stop/period)).

Reply:
OK, did not know that, thanks for the link.